packages/atomic/src/components/facets/facet-store/facet-store.ts

```diff
@@ -0,0 +1,3 @@
+import {createStore} from '@stencil/store';
+
+export const facetStore = createStore<Record<string, string>>({});
```

Comment:
Like you mentioned during the scrum, there could be multiple search interfaces in a page, that means possible state interference. There would be multiple approches here:
- Leverage the headless engine and add reducers (downside = boilerplate)
- Add a binding to the search interface for "global" interface state like the i18n & engine (personal favorite)
- have an interface store with multiple interfaces object. Put the facets in those. (perhaps the simplest solution)

Reply:
I would vote for #2, and to bind states to the central search interface.

We're using it already anyway, in a natural manner, for other things that are "stateful" and it seems to be working well ?